Makefile.in

```
@@ -7,8 +7,8 @@ KEXT_SOURCES = src/normaliz.cc
 # workaround for building with GAP 4.11 on Cygwin: we need -std=gnu++11
 # instead of -std=c++11 to avoid an error during compilation; see
 # https://github.com/gap-packages/NormalizInterface/pull/91 for details
-KEXT_CXXFLAGS = @CPPFLAGS@ @NORMALIZ_CPPFLAGS@ -std=gnu++11
-KEXT_LDFLAGS = @LDFLAGS@ -lgmp @NORMALIZ_LDFLAGS@ -lnormaliz -lstdc++
+KEXT_CXXFLAGS = @CPPFLAGS@ @NORMALIZ_CPPFLAGS@  @GMP_CPPFLAGS@ -std=gnu++11
+KEXT_LDFLAGS = @LDFLAGS@ -lgmp @NORMALIZ_LDFLAGS@ @GMP_LDFLAGS@ -lnormaliz -lstdc++
```

I am surprised this works, normally `-L` flags must come before `-l` flags they are supposed to affect, i.e., 
```suggestion
KEXT_LDFLAGS = @LDFLAGS@ @GMP_LDFLAGS@ -lgmp @NORMALIZ_LDFLAGS@ -lnormaliz -lstdc++
```